project_W/app.py

```diff
@@ -99,157 +98,123 @@ def login():
 
         if not (user and user.check_password(password)):
             logger.info(" -> incorrect credentials")
-            return jsonify(msg="Incorrect credentials provided"), 400
+            return jsonify(msg="Incorrect credentials provided", errorType="auth"), 400
 
         logger.info(" -> login successful, returning JWT token")
-        return jsonify(access_token=create_access_token(user))
+        return jsonify(msg="Login successful", access_token=create_access_token(user))
 
-    @app.post("/api/requestPasswordReset")
+    @app.get("/api/user/requestPasswordReset")
     def requestPasswordReset():
-        email = request.form['email']
+        email = request.args.get("email", type=str)
```

Comment:
This will return `None` instead of throwing if the key is missing from the arguments. Is that intentional?

Reply:
No it is not, I will fix it. BTW, the experience for when you forget to provide some arguments to a post/get request is currently a bit rough in general, the response is anything but helpful. But that is not that important I think, I don't really feel like going through all routes and add if conditions with nice responses right now :D